test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -374,6 +374,139 @@ public void RedirectToRoutePermanent_WithParameterRouteName_SetsResultSameRouteN
+        }
+
+        [Fact]
+        public void Created_WithUriParameter_SetsCreatedLocation()
+        {
+            // Arrange
+            var controller = new Controller();
+            var uri = new Uri("http://test/url");
+
+            // Act
+            var result = controller.Created(uri, null);
+
+            // Assert
+            Assert.IsType<CreatedResult>(result);
+            Assert.Same(uri.AbsoluteUri, result.Location);
+        }
+
+        [Fact]
+        public void CreatedAtAction_WithParameterActionName_SetsResultActionName()
+        {
+            // Arrange
+            var controller = new Controller();
+
+            // Act
+            var result = controller.CreatedAtAction("SampleAction", null);
+
+            // Assert
+            Assert.IsType<CreatedAtActionResult>(result);
+            Assert.Equal("SampleAction", result.ActionName);
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        [InlineData("SampleController")]
+        public void CreatedAtAction_WithActionControllerAndNullRouteValue_SetsSameValue(
+            string controllerName)
+        {
+            // Arrange
+            var controller = new Controller();
+
+            // Act
+            var result = controller.CreatedAtAction("SampleAction", controllerName, null, null);
+
+            // Assert
+            Assert.IsType<CreatedAtActionResult>(result);
+            Assert.Equal("SampleAction", result.ActionName);
+            Assert.Equal(controllerName, result.ControllerName);
+        }
+
+        [Theory]
+        [MemberData(nameof(CreatedAndRedirectTestData))]
+        public void CreatedAtAction_WithActionControllerRouteValues_SetsSameValues(
+            object routeValues,
+            IEnumerable<KeyValuePair<string, object>> expected)
+        {
+            // Arrange
+            var controller = new Controller();
+
+            // Act
+            var result = controller.CreatedAtAction("SampleAction", "SampleController", routeValues, null);
+
+            // Assert
+            Assert.IsType<CreatedAtActionResult>(result);
+            Assert.Equal("SampleAction", result.ActionName);
+            Assert.Equal("SampleController", result.ControllerName);
+            Assert.Equal(expected, result.RouteValues);
+        }
+
+        [Fact]
+        public void CreatedAtRoute_WithParameterRouteName_SetsResultSameRouteName()
+        {
+            // Arrange
+            var controller = new Controller();
+            var routeName = "SampleRoute";
+
+            // Act
+            var result = controller.CreatedAtRoute(routeName, null);
+
+            // Assert
+            Assert.IsType<CreatedAtRouteResult>(result);
+            Assert.Same(routeName, result.RouteName);
+        }
+
+        [Theory]
+        [MemberData(nameof(CreatedAndRedirectTestData))]
+        public void CreatedAtRoute_WithParameterRouteValues_SetsResultSameRouteValues(
```

Comment:
Same comment, I don't see any value in a `[Theory]` for a test that just sets a property.